src/main/java/org/swisspush/redisques/handler/RedisquesHttpRequestHandler.java

```diff
@@ -325,7 +354,7 @@ private void bulkPutLocks(RoutingContext ctx, JsonArray locks) {
                         respondWith(StatusCode.OK, ctx.request());
                     } else {
                         String errorType = reply.result().body().getString(ERROR_TYPE);
-                        if (errorType != null && BAD_INPUT.equalsIgnoreCase(errorType)) {
```

Comment:
null-Check not necessary. `String.equalsIgnoreCase(..)` allows `null` as Parameter